cmd/spiffe-helper/main.go

```diff
@@ -58,3 +69,41 @@ func startSidecar(configFile string, daemonModeFlag bool, log logrus.FieldLogger
 	log.Info("Launching daemon")
 	return spiffeSidecar.RunDaemon(ctx)
 }
+
+func startHealthServer(configFile string, daemonModeFlag bool, log logrus.FieldLogger) error {
+	hclConfig, err := config.ParseConfig(configFile)
+	if err != nil {
+		return fmt.Errorf("failed to parse %q: %w", configFile, err)
+	}
+	hclConfig.ParseConfigFlagOverrides(daemonModeFlag, daemonModeFlagName)
+	if err := hclConfig.ValidateConfig(log); err != nil {
+		return fmt.Errorf("invalid configuration: %w", err)
+	}
```

Comment:
This is duplicated from lines 50-59 above

Reply:
Now that this is in another package, might make sense to keep.